io/copy_n.ts

```diff
@@ -24,7 +24,9 @@ const DEFAULT_BUFFER_SIZE = 32 * 1024;
  * @param size Read size
  * @returns Number of bytes copied
  *
- * @deprecated This will be removed in 1.0.0. Use the {@link https://developer.mozilla.org/en-US/docs/Web/API/Streams_API | Web Streams API} instead.
+ * @deprecated Pipe through
+ * {@linkcode https://jsr.io/@std/streams/doc/~/ByteSliceStream | ByteSliceStream}
```

Comment:
Similarly to the case of #5990, I think it would be better to mention how to convert Reader/Writer to ReadableStream/WritableStream. Otherwise some users would be stuck with type errors.